src/components/modals/SettingsModal/SettingsDialog.tsx

```
@@ -63,14 +64,44 @@
         type: "text",
     },
     {
-        helperText: "[JSON] Key to extract the log level from.",
+        helperText: (
+            <span>
+                [Structured] Key that maps to each log event's log level. See
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)